styles.css

```diff
@@ -0,0 +1,550 @@
+}
+
+.canopy-list {
+  display: flex;
+  flex-direction: row;
+  gap: 30px;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.resume {
+  width: 192px;
+  margin-left: 0;
+}
+
+.about {
+  display: grid;
+  gap: 80px;
+  border-top-right-radius: 30px;
+  background: #fff;
+  padding: 20px;
+}
+
+.about-content {
+  padding-top: 41px;
+  display: flex;
+  flex-direction: column;
+}
+
+.connect-media {
+  display: flex;
+  list-style: none;
+  justify-content: flex-start;
+  gap: 18px;
+  margin-top: -4px;
+  margin-left: -42px;
+}
+
+.about-list {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  padding: 24px;
+  margin-top: -17px;
+}
+
+.languages li {
+  list-style: none;
+  background-color: rgb(247, 247, 249);
+  width: 103px;
+  height: 97px;
+  border-radius: 8px;
+  padding: 9px 12px;
+}
+
+.languages img {
+  height: 50px;
+  margin: 9px;
+  padding: 0;
+}
+
+h5 {
+  margin-top: -2px;
+  font-size: 15px;
+  font-weight: 500;
+}
+
+.listItem {
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #253858;
+}
+
+.listItem img {
+  float: right;
+}
+
+.home-content h2 {
+  color: #fff;
+  text-align: center;
+}
+
+.contact-intro {
+  color: #fff;
+}
+
+hr {
+  width: 100%;
+  margin-top: 30px;
+  margin-bottom: 26px;
+}
+
+/* Footer css */
+.contact_section {
+  background: #6070ff;
+  background-repeat: no-repeat;
+  padding-bottom: 109px;
+  border-top-left-radius: 80px;
+  height: 100vh;
+}
+
+.footer_content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: url("images/contact-bg-image.png");
+  margin: -88px 0 0 0;
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: top right;
+}
+
+input {
+  height: 26px;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  gap: 20px;
+  margin-top: 35px;
+}
+
+form input,
+textarea {
+  border-radius: 8px;
+  border: solid 1px #cfd8dc;
+  padding: 15px;
+  font-size: 1rem;
+}
+
+textarea {
+  height: 131px;
+}
+
+.contact-header {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 44px;
+  color: #fff;
+  margin-top: 85px;
+  margin-bottom: 6px;
+  align-self: center;
+}
+
+.contact-message {
+  font-size: 20px;
+  line-height: 28px;
+  font-weight: 400;
+  text-align: center;
+  color: #ebebff;
+  margin: 12px 35px;
+}
+
+.submit {
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+}
+
+.see-more:hover {
+  color: white;
+  background-color: #6070ff;
+}
+
+.see-more:active {
+  color: white;
+  background-color: #2230d2;
+}
+
+@media screen and (min-width: 768px) {
+  /* ---------Header section----------- */
+  .menu_mobile {
+    display: none;
+  }
+
+  .logo {
+    margin-left: 5%;
+  }
+
+  .menu_desktop {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 5%;
+    background: #fff;
+    position: fixed;
+    width: 100%;
+  }
```

Comment:
The margin between your navbar and the section below is not correct. Please fix it.